test/e2e/firewall-rules.e2e.ts

```diff
@@ -181,7 +184,10 @@ test('firewall rule form targets table', async ({ page }) => {
   // add a subnet by selecting from a dropdown; make sure 'mock-subnet' is present in the dropdown,
   // even though VPC:'mock-subnet' has already been added
   await selectOption(page, 'Target type', 'VPC subnet')
+  // addButton should be disabled again, as type has changed but no value has been entered
+  await expect(addButton).toBeDisabled()
   await selectOption(page, subnetNameField, 'mock-subnet')
+  await expect(addButton).toBeEnabled()
```

Comment:
This is technically covered by the `click()` and `expectRowVisible` on the next two lines, but I think having this line here (along with the `expect…Disabled` two lines up) makes it clear that the `selectOption` on the line in the middle of them is what enabled the button.